templates/taxi/car_list.html

```diff
@@ -1,7 +1,10 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>
+    Car list |
+    <a href="{% url 'taxi:car-create' %}">CREATE</a>
+  </h1>
```

Comment:
The only thing I don't like is the size of this word. Try to make it more beautiful, please

Reply:
![image](https://user-images.githubusercontent.com/111114742/200575773-eaac6d46-35a4-4f57-9d42-baae396890ea.png)

Reply:
![image](https://user-images.githubusercontent.com/111114742/200577514-4d0c5ce9-2a65-41d5-8e40-86d195a1108d.png)